### ISSUE TITLE
[Design, FE] 10-10EZ Short-form: Review Household Information tab ordering > [Other] Reading and tab order may hide information from screen reader users. (00.00.3)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#40498]
#### VFS team name
10-10 Team

#### VFS product name
Health Care Application 10-10EZ

#### VFS feature name
10-10EZ Short Form (Auto-qualifiers)

#### Point of Contact/Reviewers
Brian DeConinck (@briandeconinck) - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
No VA.gov Experience Standard for the issue found.

### Issue Details
Step 4 of 6 [disability rating <50%]: Household Information, when a user selects "No" for "Do you want to provide your financial information?" an info alert appears notifying the user that if they don't provide financial information that will prevent them from enrolling.

For a screen reader user navigating through the form, the natural next step after selecting "No" is to press [tab] to jump to the next form field or other interactive element. This will jump them over the info alert and straight to "Finish this application later," meaning they will miss some important information.

### Link, screenshot or steps to recreate
1. Log into the RI as user 11
2. Complete the form up to Step 4 of 6.
3. Activate a screen reader (recommend NVDA on Windows or VoiceOver on Mac for testing).
4. Tab to the radio buttons for "Do you want to provide your financial information?" and use the arrow keys to select "No."
5. Press [tab] to move to the next interactive element on the page.
6. Note that the info alert text is never announced by the screen reader.

### VA.gov Experience Standard
[Category Number 00, Issue Number 00](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References

---

### Platform Recommendation
Consider moving the alert before the question, and removing the conditional logic that makes it appear when No is selected. It's information that's relevant to deciding which radio button to choose, so it should be presented before the user has to make that decision.

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.

**Comment by shiragoodman** [comment]
@briandeconinck can you please update the title of this ticket?

**Comment by hdjustice** [comment]
In Production - skipped in Shortform flow
May need to set a focus point or something so that the screen-reader will catch it.
Household section may be revisited for redesign - Low priority

**Comment by shiragoodman** [comment]
@hdjustice Can you clarify what you mean by "in production" ? Do you mean that the issue was found in production or the fix has been implemented in production? If it is the former and the issue has not yet been resolve, please open this ticket back up. We request that you only close issue tickets that have been addressed/fixed.

**Comment by shiragoodman** [comment]
again, thank you for clarifying!
